docs/guides/device-gateway/arm64.md

```diff
@@ -0,0 +1,163 @@
+
+In this guide, you'll learn how to install ngrok on any Linux ARM64 device to forward traffic from public endpoints to your upstream services or for remote management. You'll also create some a traffic policy to protect your device from unauthorized traffic.
+
+This guide is specific to ARM64 devices on Linux—if you're using a different device or CPU architecture, first check whether your platform meets the ngrok agent's [system and resource requirements](/docs/agent/index.mdx##system-requirements). We also have other guides that may fit your use case more precisely:
+
+- [Linux AMD64 devices](/docs/guides/device-gateway/linux.md)
+- [Raspberry Pi](/docs/guides/device-gateway/raspberry-pi.md)
+- [Raspberry Pi OS](/docs/guides/device-gateway/raspbian.md)
+
+## Prerequisites
+
+To follow this guide, you need:
+
+- An [ngrok account](https://dashboard.ngrok.com/signup) (some features require a [pay-as-you-go plan](https://ngrok.com/pricing))
+- Any ARM64 device running Linux (see our [system requirements](/docs/agent/index.mdx##system-requirements) for all supported architectures)
+
+## Step 1: Install the ngrok Agent
+
+1. Open a shell session with your remote ARM64 device.
+
+2. Download the latest ngrok agent binary for ARM64 devices.
+
+   ```bash
+   wget https://bin.equinox.io/c/bNyj1mQVY4c/ngrok-v3-stable-linux-arm64.tgz
+   ```
+
+3. Unzip the file and move it to a directory in your `PATH`, like `/usr/local/bin`.
+
+   ```bash
+   sudo tar xvzf ./ngrok-v3-stable-linux-amd64.tgz -C /usr/local/bin
+   ```
+
+4. Link the ngrok agent to your ngrok account with your authtoken, replacing `{NGROK_AUTHOKEN}` with the value found in your [ngrok dashboard](https://dashboard.ngrok.com/get-started/your-authtoken).
+
+   ```bash
+   ngrok authtoken {NGROK_AUTHTOKEN}
+   ```
+
+## Step 2: Enable SSH access
+
+If you want to perform remote administration on your ARM64 device using a reserved TCP address and an encrypted tunnel, instead of relying on IP addresses, you can create a [TCP tunnel](/docs/tcp/index.mdx) at port `22` on your ARM64 device.
+
+1. Start the TCP tunnel with `ngrok`.
+
+   ::::warning
+   TCP endpoints are only available on a free plan after [adding a valid payment method](https://dashboard.ngrok.com/settings#id-verification) to your account. If you get a different error, ensure your authtoken is configured correctly.
+   ::::
+
+   ```bash
+   ngrok tcp 22
+   ```
+
+   When the ngrok agent starts, you'll see a `Forwarding` line with similar information about the public endpoint for the TCP tunnel to your device:
+
+   ```bash
+   Forwarding                    tcp://4.tcp.us-cal-1.ngrok.io:16420 -> localhost:22
+   ```
+
+1. On your local workstation, SSH into your ARM64 device with the command below, replacing the variables with the following:
+
+   - `{NGROK_PORT}`: The port number assigned to your ngrok agent (e.g. if the agent shows `tcp://4.tcp.us-cal-1.ngrok.io:12345`, your port number is `12345`).
```

Comment:
@joelhans I slacked you about this formatting. It's not a deal breaker for me though.